.github/ISSUE_TEMPLATE/03-reportmissinggame.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Missing Game
+about: Report a game that was not correctly detected by the app
+title: ''
+labels: 'user raised'
+assignees: ''
+
+---
+
+# Missing Game Report
+
+## Name
+<!-- Name of the game -->
+
+## Store
+<!-- Game store it was purchased from (Steam/GOG/Epic/Xbox/etc) -->
+
+## Operating System
+<!-- Windows/Linux/Steam Deck. For Linux, please provide the specific Linux Distro -->
```

Comment:
Should tell the user to upload logs.

Reply:
https://github.com/Nexus-Mods/templates-test/issues/new?template=ReportMissingGame.yml
^ This was my original design for this (to use a form) but forms appear above templates so I turned it into plain text.

Reply:
Added this to the template.